src/App.js

```diff
@@ -1,24 +1,43 @@
-import logo from './logo.svg';
+import { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import { Toaster } from 'react-hot-toast';
+import { currentUser } from './redux/auth/currentUserSlice';
 import './App.css';
+import Login from './components/auth/Login';
+import SignUp from './components/auth/SignUp';
+import Home from './components/home/Home';
+import CarsIndex from './components/cars/CarIndex/CarIndex';
+import CarShow from './components/cars/CarShow/CarShow';
+import Layout from './components/layout/Layout';
+import AddNewCar from './components/cars/AddNewCar/AddNewCar';
+import AddReservation from './components/reservation/AddReservation';
+import ReservationList from './components/reservation/ReservationList';
+import CarDelete from './components/cars/CarDelete/CarDelete';
 
 function App() {
```

Comment:
- [ ] We got still 3 functions to refactor to ES6:

![image](https://github.com/muneeb3123/Fast-Lane-Racers-front-end/assets/57421823/d800715c-30a9-49b0-a871-2a807fdc69c2)

Should take less than a minute. 👍